src/server/logging/do_tags_match.js

```diff
@@ -0,0 +1,24 @@
+import { isArray } from 'lodash';
+
+export function doTagsMatch(event, expectedTags) {
+  if (!event || !isArray(event.tags)) {
+    return false;
+  }
+
+  if (event.tags.length !== expectedTags.length) {
+    return false;
+  }
+
+  const unmatchedEventTags = event.tags.slice(0);
+  const unmatchedExpectedTags = [];
```

Comment:
I am thinking we don't even need to sort. The expected tags would need to match the order of the events tags. Trying to not do more than we need to.